Python/ceval.c

```diff
@@ -2124,6 +2124,16 @@ _PyEval_UnpackIterableStackRef(PyThreadState *tstate, _PyStackRef v_stackref,
         _PyErr_Format(tstate, PyExc_ValueError,
                       "too many values to unpack (expected %d)",
                       argcnt);
+        PyObject *exc = _PyErr_GetRaisedException(tstate);
+        PyObject_SetAttr(exc,
+                         PyUnicode_FromString("_is_incomplete_unpack_errmsg"),
```

Comment:
The “flag attribute” is not necessary; you can test for the presence of `_expected_arg_count` itself. Perhaps put `unpack` in that name.